unitTests/controllers/DynamicFacetRangeQueryControllerTest.ts

```diff
@@ -61,13 +61,13 @@ export function DynamicFacetRangeQueryControllerTest() {
       });
     });
 
-    describe('when there no values', () => {
+    describe('when calling reset', () => {
```

Comment:
Are we losing anything by completely removing that test ?

Should we be testing the "empty" case (no values/empty arrays) and check that freeze is set properly ?

Reply:
Good point, I added another test to capture that case 💯